utilities/backupable/backupable_db_test.cc

```diff
@@ -1505,41 +1515,6 @@ TEST_F(BackupableDBTest, TableFileCorruptedBeforeBackup) {
   CloseDBAndBackupEngine();
 }
 
-// Test if BackupEngine will fail to create new backup if some blob files has
-// been corrupted and the blob file checksum is stored in the DB manifest
-TEST_F(BackupableDBTest, BlobFileCorruptedBeforeBackup) {
-  const int keys_iteration = 50000;
-  options_.enable_blob_files = true;
-
-  OpenDBAndBackupEngine(true /* destroy_old_data */, false /* dummy */,
-                        kNoShare);
-  FillDB(db_.get(), 0, keys_iteration);
-  CloseAndReopenDB(/*read_only*/ true);
-  // corrupt a random blob file in the DB directory
-  ASSERT_OK(CorruptRandomDataFileInDB(kBlobFile));
-  // file_checksum_gen_factory is null, and thus blob checksum is not
-  // verified for creating a new backup; no correction is detected
-  ASSERT_OK(backup_engine_->CreateNewBackup(db_.get()));
-  CloseDBAndBackupEngine();
-
-  // delete old files in db
-  ASSERT_OK(DestroyDB(dbname_, options_));
-
-  // Enable file checksum in DB manifest
-  options_.file_checksum_gen_factory = GetFileChecksumGenCrc32cFactory();
-  OpenDBAndBackupEngine(true /* destroy_old_data */, false /* dummy */,
-                        kNoShare);
-  FillDB(db_.get(), 0, keys_iteration);
-  CloseAndReopenDB(/*read_only*/ true);
-  // corrupt a random blob file in the DB directory
-  ASSERT_OK(CorruptRandomDataFileInDB(kBlobFile));
-
-  // file checksum is enabled so we should be able to detect any
-  // corruption
-  ASSERT_NOK(backup_engine_->CreateNewBackup(db_.get()));
```

Comment:
I think we should keep these two test cases; removing them would decrease coverage since their counterparts corrupt table files specifically. To reduce code duplication, the body of the test method could be moved to a helper that takes a `FileType` parameter; the helper could then be called from `TableFileCorruptedBeforeBackup` / `BlobFileCorruptedBeforeBackup`.